depends/common/openssl/CMakeLists.txt

```diff
@@ -12,69 +14,55 @@ if(CORE_SYSTEM_NAME MATCHES "android")
   elseif(CPU MATCHES i686)
     set(OPENSSL_TARGET android-x86)
   endif()
-  set(configure_command CC=${CMAKE_C_COMPILER} AR=${CMAKE_AR} RANLIB=${CMAKE_RANLIB} INSTALL_PREFIX="" ANDROID_NDK=${TOOLCHAIN}
-                        <SOURCE_DIR>/Configure no-shared zlib
-                        --prefix=${OUTPUT_DIR}
-                        --with-zlib-include=${OUTPUT_DIR}/include
-                        --with-zlib-lib=${OUTPUT_DIR}/lib
-                        ${OPENSSL_TARGET} ${CMAKE_C_FLAGS} -I${OUTPUT_DIR}/include
-      && sed -ie "s|apps test||" ${PROJECT_SOURCE_DIR}/Makefile)
-endif()
-
-if(CORE_SYSTEM_NAME MATCHES "linux" OR CORE_SYSTEM_NAME MATCHES "rbpi")
-  if(CMAKE_TOOLCHAIN_FILE)
-    if(CPU MATCHES aarch64 OR CPU MATCHES arm64 OR CPU STREQUAL x86_64 OR ARCH MATCHES aarch64 OR ARCH STREQUAL x86_64)
-      set(bitness 64)
-    elseif(CPU MATCHES arm OR CPU MATCHES "i.86" OR ARCH MATCHES arm OR CPU MATCHES cortex-a7 OR CPU MATCHES arm1176jzf-s)
-      set(bitness 32)
-    else()
-      message(WARNING "Could not detect bitness for CPU ${CPU} - assuming 32bit")
-      set(bitness 32)
-    endif()
-
-    set(configure_command CC=${CMAKE_C_COMPILER} AR=${CMAKE_AR} INSTALL_PREFIX=""
-                          <SOURCE_DIR>/Configure no-shared zlib
-                          --prefix=${OUTPUT_DIR}
-                          --with-zlib-include=${OUTPUT_DIR}/include
-                          --with-zlib-lib=${OUTPUT_DIR}/lib
-                          linux-generic${bitness} ${CMAKE_C_FLAGS} -I${OUTPUT_DIR}/include
-        && sed -ie "s|apps test||" ${PROJECT_SOURCE_DIR}/Makefile)
+elseif(CORE_SYSTEM_NAME MATCHES "linux" OR CORE_SYSTEM_NAME MATCHES "rbpi")
+  if(CPU MATCHES aarch64 OR CPU MATCHES arm64 OR CPU STREQUAL x86_64 OR ARCH MATCHES aarch64 OR ARCH STREQUAL x86_64)
+    set(OPENSSL_TARGET linux-generic64)
+  elseif(CPU MATCHES arm OR CPU MATCHES "i.86" OR ARCH MATCHES arm OR CPU MATCHES cortex-a7 OR CPU MATCHES arm1176jzf-s)
+    set(OPENSSL_TARGET linux-generic 32)
```

Comment:
is this correct? the space between linux_generic and 32?

Reply:
no it isn't